CHANGELOG.md

```diff
@@ -2,13 +2,53 @@
 
 ## Unreleased
 
+### New features
+
+#### Import GOV.UK Frontend JavaScript as ES modules
+
+You can now import GOV.UK Frontend's component JavaScript into your service as ES modules, if you're using a bundler.
+
+This allows you to import only the JavaScript you need, and helps reduce duplication of things like polyfills.
+
+Because we are shipping ES modules in addition to how we currently publish our component JavaScript, this change is
```

Comment:
```suggestion
As we're shipping ES modules in addition to how we currently publish our component JavaScript, this change is
```
P.S. Noob question: what does "in addition to" mean here?

Reply:
Not a noob question at all. We currently ship our JavaScript in one format (called UMD). This PR introduces a second way of shipping our JavaScript (in a different format, called ESM). What I was trying to say here is that we will continue to ship our JavaScript as UMD. The ESM format is *in addition to* what we're already doing, rather than replacing it.

Reply:
Ahh I see - ta!

I wonder will we have any users who'll be using our release notes for the first time, and are unaware of how we've done things before.

First go at wording tweak:

> As we're now shipping ES modules in addition to how we already ship JavaScript (using UMD, or Universal Module Definition), this change is backwards compatible.